Corale.Colore/Core/Device.cs

```diff
@@ -40,7 +40,7 @@ public abstract class Device : IDevice
         /// <summary>
         /// Clears the current effect on the device.
         /// </summary>
-        public void Clear()
+        public virtual void Clear()
         {
             SetAll(Color.Black);
```

Comment:
My only comment was about this line.  Should we be setting Black as clear for all devices that don't override the method?  I can just see us adding a new device type in the future and having a similar issue by forgetting to override.